### ISSUE TITLE
Add documentation for Labs program plan

### ISSUE THREAD

**Issue by MaxWofford** [hackedu/meta#140]
The specific implementation of this was discussed in a one on one between @zachlatta and I.

EDIT: The documentation for the Labs program should include the format for the labs program

```md
1. Workshops
   - These are based on the workshops from Hack Camp
3. Case Studies
   - To collect data from each meeting without core hackEDU team members being
     there, each club meeting will have a case study
   - Each case study consists of 2 files:
     1. planning.md
     2. recap.md
        ex. https://github.com/hackedu/hackedu/blob/e1b9dd8e6c35d5dcc799340794d46439e6e89f9b/case_studies/cherry_hill_high_school_east/02_first_meeting/recap.md
```

**Comment by zachlatta** [comment]
This should probably go in `hackedu/hackedu` in the `meta/labs` directory.

**Comment by zachlatta** [comment]
Can you also include the implementation we discussed in the issue
description?

On Fri, Sep 11, 2015 at 4:07 PM, Max Wofford <notifications@github.com>
wrote:

> The specific implementation of this was discussed in a one on one between
> @zachlatta <https://github.com/zachlatta> and I.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/meta/issues/140>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by MaxWofford** [comment]
@zachlatta Added. Thanks :)

**Comment by zachlatta** [comment]
Closed by #204.

**Comment by MaxWofford** [comment]
Clarification: https://github.com/hackedu/hackedu/pull/204
